mantidimaging/core/io/saver.py

```diff
@@ -8,6 +8,8 @@
 
 import h5py
 from pathlib import Path
+
+import numpy
```

Comment:
This extra import can be removed, and the code can use `np.bytes_`. (Its the `string_` to `bytes_` change that is important).